admin_guide/managing_networking.adoc

```diff
@@ -226,6 +226,15 @@ specified remote server, using a private source IP address that is not used for
 anything else. The service allows pods to talk to servers that are set up
 to only allow access from whitelisted IP addresses.
 
+[IMPORTANT]
+====
+The egress router is not intended for every outgoing connection. Creating large
+numbers of egress routers, for example, creating an egress router for every
+project or application, can push the limits of your network hardware, such as
+the number of local MAC addresses that the network interface can handle before
+falling back to filtering MAC addresses in software.
+====
```

Comment:
Seems like a bit of a run-on sentence. Maybe something like:
"Creating large numbers of egress routers, can push the limits of your network hardware. For example, creating an egress router for every project or application could exceed the number of local MAC addresses that the network interface can handle before falling back to filtering MAC addresses in software."